### PR TITLE
WMCO 3.1.1 RNs

### PR DESCRIPTION
**Pull request by codyhoag**
Release date set for Dec 7, 2021

Preview: https://deploy-preview-38821--osdocs.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-3-x.html#wmco-3-1-1

### CODE REVIEW

**Comment by codyhoag** [comment]
@sgaoshang please verify the release notes for the WMCO 3.1.1 release. Thanks!

File: windows_containers/windows-containers-release-notes-3-x.adoc [hunk 1]
```diff
@@ -17,6 +17,26 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
...
+== Release notes for Red Hat Windows Machine Config Operator 3.1.1
+
+Issued: 2021-12-07
+
+The WMCO 3.1.1 is now available with bug fixes. The components of the WMCO were released in link:https://access.redhat.com/errata/RHBA-2021:4710[RHBA-2021:4710].
+
+[id="wmco-3-1-1-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `windows-exporter` metrics endpoint object contained a reference to a deleted machine. This incorrect reference caused the WMCO to ignore delete events for machines with invalid IP addresses. This bug fix removes the validation of the machine object from the event filtering, allowing the `windows-exporter` metrics endpoint object to correctly update when the machine is still in the `Deleting` phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008994[**BZ#2008994**])
```
Comment (sagidlow) [inline windows_containers/windows-containers-release-notes-3-x.adoc:30]:
```suggestion
* Previously, the `windows-exporter` metrics endpoint object contained a reference to a deleted machine. This incorrect reference caused the WMCO to ignore deleted events for machines with invalid IP addresses. This bug fix removes the validation of the machine object from the event filtering, allowing the `windows-exporter` metrics endpoint object to correctly update when the machine is still in the `Deleting` phase. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2008994[**BZ#2008994**])
```

Assuming this was intended to be "deleted" and not "delete". If my suggestion is wrong, please ignore.

**Review by sagidlow — COMMENTED** [review]
Just one fix potentially needed, otherwise LGTM. :)

**Comment by sgaoshang** [comment]
@codyhoag /lgtm, thanks.

**Pull request merged**